README.md

```diff
@@ -337,54 +337,54 @@ You can then use the `Slugify::slugify()` method in your controllers:
 $url = Slugify::slugify("welcome to the homepage");
 ```
 
-### Zend Framework 2
+### Laminas
 
-Slugify can be easely used in Zend Framework 2 applications. Included bridge provides a service and a view helper
+Slugify can be easily used in Laminas applications. Included bridge provides a service and a view helper
 already registered for you.
 
 Just enable the module in your configuration like this.
 
 ```php
-return [
+return array(
```

Comment:
Why is this change being made?

Reply:
These changes were made in the draft by user JustInTime. I have derived from this branch. => https://github.com/cocur/slugify/pull/261
But I'll change that again